OrbitCore/Capture.cpp

```diff
@@ -230,18 +225,6 @@ void Capture::NewSamplingProfiler() {
       std::make_shared<SamplingProfiler>(Capture::GTargetProcess);
 }
 
-//-----------------------------------------------------------------------------
-void Capture::RegisterZoneName(uint64_t a_ID, const char* a_Name) {
-  GZoneNames[a_ID] = a_Name;
-}
-
-//-----------------------------------------------------------------------------
-void Capture::AddCallstack(CallStack& a_CallStack) {
-  ScopeLock lock(GCallstackMutex);
-  Capture::GCallstacks[a_CallStack.Hash()] =
```

Comment:
Should GCallstacks also be removed?

Reply:
Interesting - apparently CaptureWindow uses it to select a callstack. It does not work now (because the map is empty) - but I wonder if it should. I will take a closer look. As it is now - it seems like this is unused as well.

Reply:
It looks like this is a bug - at some point we lost this map and callstack selection stopped working. I am investigating,

Reply:
this is not a regression from 1.50